spec/unit/application/device_spec.rb

```diff
@@ -75,30 +75,39 @@
     end
 
     it "should set waitforcert to 0 with --onetime and if --waitforcert wasn't given" do
+      Puppet[:certname] = 'device.example.com'
```

Comment:
Would this be better in a describe block with a begin block?

Reply:
Updated